app/api/attendees.py

```diff
@@ -23,8 +23,8 @@ class AttendeeListPost(ResourceList):
 
     def before_post(self, args, kwargs, data):
         require_relationship(['ticket', 'event'], data)
-        if not has_access('is_coorganizer', event_id=data['event']):
-            raise ForbiddenException({'source': 'event_id'}, "Access Forbidden")
+        if not has_access('is_user_itself', id=current_identity.id):
```

Comment:
Didn't get this. This check is basically translating as "Is the logged in user is same as the logged in user?"

Reply:
@poush yes. Its basically checks logged in like jwt_required.